node/cli/src/chain_spec.rs

```diff
@@ -56,13 +59,13 @@ struct Allocation {
     vesting: Vec<(String, BlockNumber, BlockNumber, String)>,
 }
 
-fn get_lockdrop_participants_allocation() -> Result<Allocation>{
-	let path = Path::new("node/cli/lockdrop/lockdrop-allocation.json");
-	let mut file = File::open(&path).unwrap();
-	let mut data = String::new();
-	file.read_to_string(&mut data).unwrap();
-	let a: Allocation = serde_json::from_str(&data)?;
-	return Ok(a);
+fn get_lockdrop_participants_allocation() -> Result<Allocation> {
+    let path = Path::new("node/cli/lockdrop/lockdrop-allocation.json");
+    let mut file = File::open(&path).unwrap();
+    let mut data = String::new();
+    file.read_to_string(&mut data).unwrap();
+    let a: Allocation = serde_json::from_str(&data)?;
+    return Ok(a);
```

Comment:
Are the whitespace changes intentional?

Reply:
Nope, guess my editor got ahead of itself!